recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.18.0.bb

```diff
@@ -93,18 +91,22 @@ PACKAGECONFIG[zbar]            = "-Dzbar=enabled,-Dzbar=disabled,zbar"
 #   winscreencap wpe x265
 
 EXTRA_OEMESON += " \
+    -Ddoc=disabled \
     -Ddecklink=enabled \
     -Ddvb=enabled \
     -Dfbdev=enabled \
     -Dipcpipeline=enabled \
     -Dnetsim=enabled \
     -Dshm=enabled \
+    -Dv4l2codecs=enabled \
```

Comment:
Those ought to be new PACKAGECONFIG options, no? also, in a new commit.

Reply:
This don't have any dependencies on external packages.
But it can be added as PACKAGECONFIG options